src/types.ts

```diff
@@ -16,3 +16,7 @@ export type Register = {
   password: string
   email: string
 }
+
+export type Apps = {
+  [property: string]: string
```

Comment:
🤔 Since this is mainly app names, have you considered `App[]`? We will likely have an `App` type in the future, so this may conflict with what one would expect from a plural of those.

Reply:
That said, I’ll leave it to @icidasset and @dholms to weigh in here; they’re deeper in the TS client than I am (it’s been a _while_)

Reply:
I would do something along the lines of:
```js
type App = { domain: string }
type Apps = Array<App> // sorted by domain
```

Reply:
And maybe remove the `Apps` type entirely, and just use `Array<App>` instead.

Reply:
Thanks guys for the review! I made the type changes, hopefully thats more along of the lines of what you wanted